xbmc/utils/EGLUtils.cpp

```diff
@@ -44,7 +44,7 @@ std::set<std::string> CEGLUtils::GetExtensions(EGLDisplay eglDisplay)
   const char* extensions = eglQueryString(eglDisplay, EGL_EXTENSIONS);
   if (!extensions)
   {
-    throw std::runtime_error("Could not query EGL for extensions");
+    return {};
```

Comment:
IMO we can keep the throw here, returning non-NULL for `EGL_EXTENSIONS` on the display is mandated by the core EGL spec since the first version. It's only calling `EGL_EXTENSIONS` without a display to get client extensions that was added later on.